src/styles/_header.scss

```
@@ -37,5 +37,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition: color 0.3s ease 0s;
```

```suggestion
    transition: color 0.3s ease;
```
Это можно не указывать